Makefile

```diff
@@ -32,3 +32,36 @@ breakpad:
 
 json_enhancements_pg_extension: bootstrap
 	./scripts/json-enhancements.sh
+
+
+# Docker related rules
+
+.PHONY: dockerbuild dockersetup dockerclean dockertest dockerrun
+
+DC := $(shell which docker-compose)
+
+.docker-build:
+	make dockerbuild
+
+dockerbuild:
+	${DC} build processor
+	${DC} build webapp
+	${DC} build test
+	touch .docker-build
+
+# NOTE(willkg): We run setup in the webapp container because the webapp will own
+# postgres going forward and has the needed environment variables.
+# FIXME(willkg): These are not idempotent, but they should be. Plus we should
+# run migrations here, too.
+dockersetup: .docker-build
+	-${DC} run --entrypoint /app/docker/run_setup_postgres.sh webapp
```

Comment:
Minor nit, maybe better to use command instead of entrypoint: `-${DC} run webapp /app/docker/run_setup_postgres.sh`